### PR TITLE
refactor: Refactor template resources

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Before:
Screen Shot 2022-10-27 at 14 38 10

After:
Screen Shot 2022-10-27 at 14 38 20

### CODE REVIEW

**Comment by kylecarbs** [comment]
The button looks a bit misleading there I feel, almost as though I could click it...

**Comment by BrunoQuaresma** [comment]
@kylecarbs I felt the same about the button but I don't know how to improve that, to be honest. I asked Ted's help but I think this version is better than what we have now.

**Comment by bpmct** [comment]
> @kylecarbs I feel the same about the button but I don't know how to improve that, to be honest. I asked Ted's help but I think this version is better than what we have now.

Would a thin outline with no background work?

**Comment by BrunoQuaresma** [comment]
> Would a thin outline with no background work?

Just context, this comment was before the current screenshot where I was using buttons instead of pills - I know the difference is only the amount of border-radius 😆 - I tried without the background and I didn't like it too much, to be honest. It does not give enough contrast to that.

File: site/src/components/AppLink/AppLink.stories.tsx [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import { Story } from "@storybook/react"
-import { MockWorkspace } from "../../testHelpers/renderHelpers"
+import {
+  MockWorkspace,
+  MockWorkspaceAgent,
+  MockWorkspaceApp,
+} from "../../testHelpers/renderHelpers"
```
Comment (Kira-Pilot) [inline site/src/components/AppLink/AppLink.stories.tsx:6]:
```suggestion
} from "testHelpers/renderHelpers"
```

File: site/src/components/AppLink/AppLink.tsx [hunk 2]
```diff
@@ -19,76 +17,50 @@ export const Language = {
...
-  agentName: TypesGen.WorkspaceAgent["name"]
-  appName: TypesGen.WorkspaceApp["name"]
-  appIcon?: TypesGen.WorkspaceApp["icon"]
-  appCommand?: TypesGen.WorkspaceApp["command"]
-  appSubdomain: TypesGen.WorkspaceApp["subdomain"]
-  appSharingLevel: TypesGen.WorkspaceApp["sharing_level"]
-  health: TypesGen.WorkspaceApp["health"]
+  workspace: TypesGen.Workspace
+  app: TypesGen.WorkspaceApp
+  agent: TypesGen.WorkspaceAgent
```
Comment (Kira-Pilot) [inline site/src/components/AppLink/AppLink.tsx:22]: 😍

File: site/src/components/AppLink/AppPreviewLink.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { makeStyles } from "@material-ui/core/styles"
+import { Stack } from "components/Stack/Stack"
+import { FC } from "react"
+import * as TypesGen from "../../api/typesGenerated"
```
Comment (Kira-Pilot) [inline site/src/components/AppLink/AppPreviewLink.tsx:4]:
```suggestion
import * as TypesGen from "api/typesGenerated"
```

File: site/src/components/AppLink/ShareIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import * as TypesGen from "../../api/typesGenerated"
+import Tooltip from "@material-ui/core/Tooltip"
+
+export interface ShareIconProps {
+  app: TypesGen.WorkspaceApp
+}
+
+export const ShareIcon: FC<ShareIconProps> = ({ app }) => {
+  let shareIcon = <LockOutlinedIcon />
+  let shareTooltip = "Private, only accessible by you"
```
Comment (Kira-Pilot) [inline site/src/components/AppLink/ShareIcon.tsx:14]: :D translation would be nice here

Comment (BrunoQuaresma) [reply site/src/components/AppLink/ShareIcon.tsx:14]: Doing it right now

File: site/src/components/Resources/AgentRowPreview.tsx [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+            <span className={styles.agentDataValue}>{agent.name}</span>
+          </Stack>
+
+          <Stack
+            direction="row"
+            alignItems="baseline"
+            spacing={1}
+            className={combineClasses([styles.noShrink, styles.agentDataItem])}
+          >
+            <span>OS:</span>
```
Comment (Kira-Pilot) [inline site/src/components/Resources/AgentRowPreview.tsx:49]: And some translations in this file would be nice, too

**Review by Kira-Pilot — APPROVED** [review]
Just small stuff in this review; looks great!

File: site/src/components/Resources/AgentRowPreview.tsx (hunk 5 above)
Comment (BrunoQuaresma) [reply site/src/components/Resources/AgentRowPreview.tsx:49]: Sure thing!

**Pull request merged**